pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -273,6 +273,10 @@ self: super: {
     dependencies = with self; [ plenary-nvim ];
   });
 
+  gruvbox-nvim = super.gruvbox-nvim.overrideAttrs (old: {
+    dependencies = with super; [ lush-nvim ];
```

Comment:
```suggestion
    dependencies = with self; [ lush-nvim ];
```

Reply:
Right, I missed that (I'm a bit fuzzy on `super`/`self`). Thanks for spotting this. Fixed. :)